src/services/ProfileApiService.js

```diff
@@ -37,7 +37,16 @@ export async function patchProfile(username, params) {
         'Content-Type': 'application/merge-patch+json',
       },
     },
-  );
+  ).catch((error) => {
```

Comment:
So we should find a way to make this catch generic, probably, so it's reusable amongst all our API endpoints.  (And then hold the backend code to a standard error response!)

Reply:
👍